doc/Changelog

```diff
@@ -1,3 +1,7 @@
+9 February 2023: Philip
+	- Fix #825: Unexpected behavior with client-subnet-always-forward
+	  and serve-expired
+
```

Comment:
For PRs it is better to update the changelog after the merge, to avoid these conflicts and have the correct date.
(GitHub's suggestion does not seem to apply on all changed lines [1..4])
```suggestion
```